src/windows/appw.c

```diff
@@ -193,10 +193,10 @@ static void app_register_raw_input(USHORT usage_page, USHORT usage, DWORD flags,
 static void app_adjust_window_rect(MTY_App *app, float scale, RECT *r)
 {
 	if (app->AdjustWindowRectExForDpi) {
-		app->AdjustWindowRectExForDpi(r, WS_OVERLAPPEDWINDOW, FALSE, 0, lrint(scale * 96));
+		app->AdjustWindowRectExForDpi(r, WS_CAPTION | WS_BORDER, FALSE, 0, lrint(scale * 96));
 
 	} else {
-		AdjustWindowRect(r, WS_OVERLAPPEDWINDOW, FALSE);
+		AdjustWindowRect(r, WS_CAPTION | WS_BORDER, FALSE);
```

Comment:
So, we're changing from:
WS_OVERLAPPEDWINDOW = WS_OVERLAPPED | WS_CAPTION | WS_SYSMENU | WS_THICKFRAME | WS_MINIMIZEBOX | WS_MAXIMIZEBOX
With implied WS_BORDER by WS_CAPTION

To:
WS_CAPTION | WS_BORDER

So, Minimizebox, and Maximizebox are now hidden? Did you check if removing one of the flags fixes it? Does everything else still work properly?

Reply:
WS_CAPTION alone is enough for this. This does not actually interact with the window, so all we're trying to emulate here is the modifiers to the _dimensions_ of the window. 

The minimise and maximise buttons do not affect the window size